packages/components/src/color-picker/inputs.js

```diff
@@ -222,11 +223,12 @@ export class Inputs extends Component {
 			return (
 				<fieldset>
 					<legend className="screen-reader-text">
-						{ __( 'Color value in HSL' ) }
+						{ __( 'Color value in' ) }
+						<abbr title={ 'Hue Saturation Lightness' }>{ __( 'HSL' ) }</abbr>
```

Comment:
See above, I'd recommend to not use `<abbr>`.

Reply:
You should use i18n `%s` placeholders. Use `Color value in %s`. This way, translators can control the location of the `%s` in the string when they translate to local languages.